extensions/wikia/PowerUser/PowerUser.class.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * Class to manage data on wikia's PowerUsers.
+ * It implements methods to add and remove
+ * poweruser properties and log these actions
+ * in a consistent way.
+ *
+ * @package PowerUser
+ * @author Adam Karmiński <adamk@wikia-inc.com>
+ */
+
+namespace Wikia\PowerUser;
+
+use Wikia\Logger\Loggable;
+
+class PowerUser {
+	use Loggable;
```

Comment:
:+1: for using `Loggable` trait. You can consider using `getLoggerContext` method to pass:

```php
'user_id' => $iUserId,
```

which is defined in object instance so that you can avoid passing it on each call to `log`.